### ISSUE TITLE
WCAG 4.2.1 test 21: false positives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#133]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7736395/test-document-8.pdf) fails **WCAG 4.2.1 test 21** ("_Missing list element(s)_"). The algorithm raises the error for many text paragraphs on pages 14 and onwards, while these are obviously just text paragraps and not lists.

[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7736418/test-document-3.pdf) also fails **WCAG 4.2.1 test 21**. In this case the algorithm raises the error for lists that have been correctly tagged as lists.

**Comment by bdoubrov** [comment]
The algorithm detects headings as list labels, which is clearly wrong. To be fixed
